### ISSUE TITLE
gradient clip

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#489]
https://pytorch.org/docs/1.13/generated/torch.nn.utils.clip_grad_norm_.html?highlight=clip#torch.nn.utils.clip_grad_norm_

**Comment by smilesun** [comment]
see pr https://github.com/marrlab/DomainLab/pull/501

**Comment by smilesun** [comment]
it seems gradient clip makes it worse.

**Comment by smilesun** [comment]
python main_out.py --te_d=1 --tr_d 0 3 --task=mnistcolor10 --bs=16 --aname=diva --trainer=fbopt --nname=conv_bn_pool_2 --epos=2000 --es=500 --mu_init=0.00001 --gamma_y=1.0 --str_diva_multiplier_type=gammad_recon_per_pixel --grad_clip=0.5
